### PR TITLE
2.1 kernels

### PR DESCRIPTION
**Pull request by pmatulis**

### CODE REVIEW

File: en/installconfig-hwe-kernels.md [hunk 1]
```diff
@@ -58,25 +100,25 @@ perform these three configurations from the CLI.
...
+and select a kernel in the 'Default Minimum Kernel Version' field. Don't
 forget to click 'Save'.
 
 ![image](../media/default_min_hwe_kernel.png)
 
 ### Machine minimum kernel
 
-To set the minimum HWE kernel on a machine basis visit the machine's page and click
-the `Edit` button in the 'Machine summary' pane. Then select a HWE kernel in the
+To set the minimum kernel on a machine basis visit the machine's page and click
```
Comment (degville) [inline en/installconfig-hwe-kernels.md]: I think the 'machine's page' should reference the Nodes page, i.e: To set the minimum kernel on a machine basis, select a machine from the Nodes page and click...

File: en/installconfig-hwe-kernels.md [hunk 2]
```diff
@@ -58,25 +100,25 @@ perform these three configurations from the CLI.
...
 'Minimum Kernel' field and 'Save changes'.
 
 ![image](../media/min_hwe_kernel.png)
 
 ### Machine kernel during deployment
 
-To set a specific HWE kernel during deployment visit the machine's page and choose
-`Deploy` under 'Take action'. Then select a HWE kernel in the 'Default
-kernel' field. Hit 'Go' to initiate the deployment.
+To set a specific kernel during deployment visit the machine's page and choose
```
Comment (degville) [inline en/installconfig-hwe-kernels.md]: As above, maybe making it clearer that the machine's page is accessed from the Nodes view.

**Comment by pmatulis** [comment]
@degville Done

**Comment by degville** [comment]
Thanks Peter. Another great update.

**Pull request merged**